templates/UmbracoExtension/Client/public/umbraco-package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "$schema": "../../umbraco-package-schema.json",
+  "id": "UmbracoExtension",
+  "name": "UmbracoExtension",
+  "version": "0.0.0",
+  "allowPackageTelemetry": false,
```

Comment:
Should we default this to true so you opt out instead of in? 🙂
Think most people dont care and will just leave the setting as is.

Reply:
Yep no problem will do that :)